src/app.js

```diff
@@ -169,4 +164,16 @@ function createApp (config, core, backend, routeBuilder, routeBroadcaster, routi
   }
 }
 
+function getBackend (backend) {
+  try {
+    return require('./backends/' + backend)
+  } catch (err) {
```

Comment:
This will catch all errors, which can be very confusing. Suppose some module that your backend relies on throws when you require it. The error you'll get is "Backend not found", but you check and the backend is definitely, definitely there.

The solution is to first call `require.resolve` which will *only* fail if the immediate dependency couldn't be found. Then require the actual module outside of the try/catch so that any other errors will bubble up normally.

For example:

```js
function loadModule (modulePath) {
  try {
    require.resolve(modulePath)
  } catch (err) {
    throw new Error('Module "' + modulePath + '" not found')
  }

  return require(modulePath)
}
```